templates/show_post.html

```diff
@@ -46,31 +46,30 @@
 
 {% block body %}
 	<script>
-		function show_hidden(uid){
+	function show_hidden(uid) {
 			div = document.getElementById(uid);
-			if (div.style.display == "none"){
+			if (div.style.display == "none") {
 				div.style.display = "block";
-			}
-			else{
+			} else {
 				div.style.display = "none";
 			}
 		}
 
-		function post_comment(msgId,divId,cid,pid,sid){
-			var xhttp = new XMLHttpRequest();
+		function post_comment(event, msgId, divId, cid, pid, sid) {
+			let xhttp = new XMLHttpRequest();
 			console.log("ON POST COMMENT")
-			div = document.getElementById(divId);
-			msg = document.getElementById(msgId).value;
+			let div = document.getElementById(divId);
```

Comment:
Bit of housekeeping; saying `div = div = document.getElementById(divId);` without declaring div with `let` or `var` first with declare `div` as a *global* variable. globals in JS are bascilly best left avoided; If your variable is only used locally, always use `var`, `let` or `const`